Doxyfile

```diff
@@ -805,7 +805,7 @@ EXCLUDE_SYMLINKS       = NO
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories for example use the pattern */test/*
 
-EXCLUDE_PATTERNS       = */build/* */contrib/depends/*
+EXCLUDE_PATTERNS       = */src/crypto/crypto_ops_builder/ref10*
```

Comment:
just curiosity, why exclude it?

Reply:
Because the SUPERCOP ref10 is already documented outside of Monero and it causes a huge disproportionate amount of documentation to be created